inc/saitypes.h

```diff
@@ -1091,6 +1173,18 @@ typedef union _sai_attribute_value_t
 
     /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_MACSEC_SALT */
     sai_macsec_salt_t macsecsalt;
+
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_SYSTEM_PORT_CONFIG */
+    sai_system_port_config_t sysportconfig;
+
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_SYSTEM_PORT_CONFIG_LIST */
+    sai_system_port_config_list_t sysportconfiglist;
+
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_FABRIC_PORT_REACHABILITY */
+    sai_fabric_port_reachability_t reachability;
+
+    /** @validonly meta->attrvaluetype == SAI_ATTR_VALUE_TYPE_PORT_ERR_STATUS_LIST */
+    sai_port_err_status_list_t porterror;
```

Comment:
sysporterror

Reply:
The port error type applies to fabric ports in the VoQ context but can apply to front panel ports.  This is not for system port.